javalib/src/main/scala/java/net/URI.scala

```diff
@@ -35,38 +35,60 @@ final class URI(origStr: String) extends Serializable with Comparable[URI] {
   if (_fld == null)
     throw new URISyntaxException(origStr, "Malformed URI")
 
-  private val _isAbsolute = fld(AbsScheme).isDefined
-  private val _isOpaque = fld(AbsOpaquePart).isDefined
+  private val _isAbsolute = _fld(AbsScheme).isDefined
+  private val _isOpaque = _fld(AbsOpaquePart).isDefined
 
-  @inline private def fld(idx: Int): js.UndefOr[String] = _fld(idx)
+  @inline private def fld(idx: Int): String = _fld(idx).orNull
```

Comment:
Also remove this usage of `js.UndefOr`?

Reply:
Oh, I had forgotten about this PR :s Sorry about that.

There's not only this spot but also the two calls to `_fld(...).isDefined` above. I can't get rid of them without obscuring the code, at the moment. The intent is to replace them by calls to helpers similar to
https://github.com/scala-js/scala-js/blob/d02805412faeeef620e203b9e98ca966067db648/javalanglib/src/main/scala/java/lang/Utils.scala#L21-L23
in a later PR, where we make a full swipe of all remaining `js.UndefOr` operations all over the javalib.

Alternatively, I could start a `Utils.scala` helper class in this PR with only `isDefined` and `orNull`, and expand it later with more stuff.

WDYT?

Reply:
Yeah, if that is the plan, doing it later is fine.